src/scss/components/_donors.scss

```diff
@@ -0,0 +1,61 @@
+.donors_info {
+  @include grid;
+}
+
+.donors {
+  position: relative;
+  overflow-x: hidden;
+}
+
+.donors_image {
+  width: 100%;
+}
+
+.donors_description {
```

Comment:
According to the design it should be:
`@include heading-3;`
`font-weight: var(--font-med);`
`@include col(2, 7);`

Reply:
thanks, i corrected it!!